nginx-ubi/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"
+
+NGINX_VERSION=1.14
```

Comment:
@sgnn7 is this consistent with what we use in the helm chart / quick start?

Reply:
Helm: 1.15
Quick start: 1.13.6

I'd probably opt we go w/ 1.15 to match the helm chart. With that said, NGINX is incredibly stable with APIs so these differences should not matter much (if at all).

Reply:
wouldn't hurt to bump up the quick start too.

Reply:
It seems that ubi8 comes with Nginx 11.4 app stream.
In order to install 1.16 we will need a redhat subscription license (they don't provide 1.15).
Can I keep it with 11.4 at the moment?

Reply:
It should be fine w/ 1.14 but it should be added as a comment that it needs to be kept as close as possible to appliance's NGINX version.